src/Build/Definition/ProjectCollection.cs

```diff
@@ -1867,601 +1867,6 @@ public ProjectAddedToProjectCollectionEventArgs(ProjectRootElement element)
             public ProjectRootElement ProjectRootElement { get; }
         }
 
-        /// <summary>
```

Comment:
Pulled this out to a separate file due to how important it is. It needed to be updated with a few capabilities and I think it may also be useful to expose this publically.